packages/engine-server/src/markdown/remark/utils.ts

```diff
@@ -378,7 +378,10 @@ export class LinkUtils {
   }
   /**
    * Get all links from the note body
-   * Currently, just look for wiki links
+   * Currently, just look for wiki links and keep existing backlinks
```

Comment:
Are the changes in this file necessary for the bug to be fixed, or are the changes in `dnode` / `storev2` sufficient? 

The modifications here feel a little bit odd to me, because this logic is part of a utils around Markdown/Remark, i.e. note parsing.  So it feels a bit strange to also return some data from `NoteProps` metadata in the return value, versus just data parsed from the note body.

Reply:
I can move it out of `LinkUtils` no problem. I'll move it to `getLinks` of `DendronEngineV2` instead as there are multiple places where we override node.links with the results of `getLinks`